src/dotnet/ReSharperPlugin.Verify/VerifyAcceptAction.cs

```diff
@@ -69,13 +69,17 @@ public void Execute(IDataContext context, DelegateExecute nextExecute)
                 if (!HasVerifyException(result))
                     continue;
 
-                var projectFile = element.GetProjectFiles().NotNull().SingleItem().NotNull();
-                var exceptionLines = result.GetExceptionChunk(2).SplitByNewLine();
-                var (receivedFileName, verifiedFileName) = (
-                    exceptionLines.FirstOrDefault(x => x.StartsWith("Received"))?.TrimFromStart("Received: "),
-                    exceptionLines.FirstOrDefault(x => x.StartsWith("Verified"))?.TrimFromStart("Verified: "));
-                var receivedFile = (projectFile.Location.Directory / receivedFileName.NotNull("receivedFileName")).FullPath;
-                var verifiedFile = (projectFile.Location.Directory / verifiedFileName.NotNull("verifiedFileName")).FullPath;
+                var exceptionLines = result.GetExceptionChunk(2)
+                    .SplitByNewLine()
+                    //Received path and Verified path are at the end of the exception message
+                    .Reverse()
+                    .ToArray();
+                var (receivedFile, verifiedFile) = (
+                    exceptionLines.FirstOrDefault(x => x.StartsWith("Received path: "))?.TrimFromStart("Received path: "),
+                    exceptionLines.FirstOrDefault(x => x.StartsWith("Verified path: "))?.TrimFromStart("Verified path: "));
+
```

Comment:
This means older versions aren't supported, right?

Reply:
it does